app/services/doc_auth/mock/result_response.rb

```diff
@@ -166,15 +183,23 @@ def classification_info
       end
 
       def doc_auth_result_from_success
-        if success?
+        if doc_auth_success?
           DocAuth::LexisNexis::ResultCodes::PASSED.name
         else
           DocAuth::LexisNexis::ResultCodes::CAUTION.name
         end
       end
 
-      def all_doc_capture_values_passing?(doc_auth_result, id_type_supported)
-        doc_auth_result == 'Passed' &&
+      def transaction_status_from_success
+        if doc_auth_success?
```

Comment:
I wonder if this is really meant to show the result of the `transaction_status` if we should call this `transaction_status_success?`

Reply:
Though thinking about it more...this isn't supposed to include selfie, right? Maybe the name is okay as is, then, since otherwise it's not very clear what `transaction` it's referring to.